app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class attribute, which means it is shared across all instances of the `Person` class. This could lead to unintended behavior if multiple instances are expected to maintain separate lists of people. Consider using an instance attribute instead to ensure each instance has its own dictionary.